doc/syslog/Syslog_Source_IP_test_plan.md

```diff
@@ -0,0 +1,165 @@
+# Syslog Source IP Test Plan
+
+## Related documents
+
+| **Document Name** | **Link** |
+|-------------------|----------|
+| Syslog Source IP HLD | [https://github.com/nazariig/SONiC/blob/1f2f6713495d04b41d19e0f201bff72af60ef52f/doc/syslog/syslog-design.md]|
```

Comment:
@JibinBao has to be fixed once the HLD is merged to master

Reply:
yes